src/engraving/playback/filters/spannerfilter.cpp

```diff
@@ -56,13 +56,6 @@ int SpannerFilter::spannerActualDurationTicks(const Spanner* spanner, const int
         return spanner->endSegment()->tick().ticks() - spanner->tick().ticks() - 1;
```

Comment:
It might be worthwhile to check if this adjustment and the adjustments for SLUR below are also still necessary, given the fix in `isPlayable`. (The fix in `isPlayable` looks good to me.)

Reply:
Yep, the `TRILL` case is indeed unnecessary.